docs/migrations/migrations.md

```diff
@@ -15,4 +15,5 @@ the chance to adjust the schema and underlying data access layer.
 ## Graphback supported migrations
 
 - OpenAPI (RESTful server migrations)
-- Database migrations (WIP)
+- Various plugins available as part of the GraphQL compose
```

Comment:
This point needs more information to be useful.

Reply:
There is link bellow. GraphQL compose can do db/mongose/json migrations

Reply:
There is no point to duplicate info - we just put reference if people want that and we have now the solution for db based access